### PR TITLE
install orm

### LINKED ISSUE #5: add database service and orm to backend setup
- add pg database orm, with sequelize ?

related to #2

### PR DESCRIPTION
**Pull request by pejrak**
relates to #8 and closes #5

### CODE REVIEW

File: server/src/http/errorHandler.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import Debug from 'debug';
```
Comment (jCobbSK) [inline server/src/http/errorHandler.ts]: Can we be consistent with filenames so it starts with capital letter -> `ErrorHandler.ts` which contains handle function?

File: server/src/database/migrations.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+```
+
+### Run migrations
+```
+ts-node ./node_modules/.bin/typeorm migration:run
+```
+
+### Revert last migration
+```
+ts-node ./node_modules/.bin/typeorm migration:revert
```
[the remark this answers is not in the document]
Comment (pejrak) [reply server/src/database/migrations.md]: cool will add that

File: server/src/http/errorHandler.ts (hunk 1 above)
Comment (pejrak) [reply server/src/http/errorHandler.ts]: this is a module import

Comment (pejrak) [reply server/src/http/errorHandler.ts]: ahh OK, understood

**Pull request merged**